src/main/resources/application.yml

```diff
@@ -5,14 +5,22 @@ app:
     grantType: password
   token:
     url: http://localhost:8080/oauth/token
+  get:
+    single:
+      user:
+        url: http://localhost:8080/api/users
```

Comment:
It's not the bet naming approach for a long run. First of all, you're using property named "app.get.single.user.url" for every user-related call. Secondly, this way you won't fully use the potential of yaml format.

### For a solution:
We know all the URLs will have a common prefix: `http://localhost:8080` - so let's create a property with that value and re-use it:
```yaml
url:
  base: http://localhost:8080
  token: ${app.url.base}/oauth/token
  api: ${app.url.base}/api
  user: ${app.url.api}/users
```

This way if you want to change the location of server application you change it in just a single place. In addition to that you will have an easier to read list of properties.